miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -453,15 +453,15 @@ public int compare(TissueType o1, TissueType o2) {
   }
 
   @Autowired
-  private QcPassedDetailService qcpassedDetailService;
+  private DetailedQcStatusService detailedQcStatusService;
 
-  @ModelAttribute("qcPassedDetails")
-  public List<QcPassedDetail> getQcPassedDetails() throws IOException {
-    List<QcPassedDetail> list = new ArrayList<>(qcpassedDetailService.getAll());
-    Collections.sort(list, new Comparator<QcPassedDetail>() {
+  @ModelAttribute("detailedQcStatuses")
+  public List<DetailedQcStatus> getDetailedQcStatuses() throws IOException {
+    List<DetailedQcStatus> list = new ArrayList<>(detailedQcStatusService.getAll());
+    Collections.sort(list, new Comparator<DetailedQcStatus>() {
       @Override
-      public int compare(QcPassedDetail o1, QcPassedDetail o2) {
-        return o1.getDescription().compareTo(o2.getDescription());
+      public int compare(DetailedQcStatus o1, DetailedQcStatus o2) {
+        return o1.getId().compareTo(o2.getId());
```

Comment:
Would it be better to sort by status (order: true > null > false), noteRequired (false > true) and description (A > Z)?